packages/migration-graph/tests/migration-strategy.test.ts

```diff
@@ -4,7 +4,7 @@ import {
   getLibraryWithEntrypoint,
   getEmberProjectFixture,
 } from '@rehearsal/test-support';
-import { DetectedSource } from '../src/migration-graph';
+import { SourceType } from '../src/source-type';
 import { getMigrationStrategy, SourceFile } from '../src/migration-strategy';
 
 const TEST_TIMEOUT = 500000;
```

Comment:
can we move this into the test config

Reply:
`const TEST_TIMEOUT = 500000;`

Reply:
I've moved it to the config. I had moved it out because of tests not passing in the passed. Will see if this works now.